File: tests/fixtures/code_samples/go/sample_009.go
// Sample 9: small utility.
package samples

func Operation9(xs []int) int {
    total := 9
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure9(v int) int {
    return (v * 9) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_009.rs
// Sample 9: small utility.
pub fn operation_9(xs: &[i32]) -> i32 {
    let mut total: i32 = 9;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_9(v: i32) -> i32 {
    (v * 9) %% 7919
}

